cellmlmanip/model.py

```diff
@@ -267,6 +267,18 @@ def get_derivative_symbols(self):
         derivative_symbols = [v for v in self.graph if isinstance(v, sympy.Derivative)]
         return sorted(derivative_symbols, key=lambda state_var: state_var.args[0].order_added)
 
+    def get_derived_quantities(self, namespace_uri=None):
```

Comment:
I think rather than putting the namespace option in this method, I'd prefer the calling code to filter the returned list. It'll make this method simpler, and doesn't really add much complexity for the caller.